core_lib/src/tool/selecttool.cpp

```diff
@@ -35,6 +35,7 @@ void SelectTool::loadSettings()
     properties.feather = -1;
     properties.stabilizerLevel = -1;
     properties.useAA = -1;
+    mPropertyEnabled[SHOWSELECTIONINFO] = true;
```

Comment:
Value also needs to be stored and added to properties, so the user doesn't have to enable/disable every time.